tests/unit/store/entity/actions.spec.ts

```diff
@@ -31,6 +43,21 @@ describe( 'entity/actions', () => {
 			} );
 		} );
 
+		it( `commits to ${EDITABILITY_UPDATE} on successful lookup and editability resolution`, () => {
+			const isEditable = true;
+			factory.setEntityEditabilityResolver( {
+				isEditable: () => Promise.resolve( isEditable ),
+			} );
+
+			const context = {
+				commit: jest.fn(),
+			};
+
+			return ( actions[ ENTITY_INIT ] as any )( context, 'Q1' ).then( () => {
+				expect( context.commit ).toHaveBeenCalledWith( EDITABILITY_UPDATE, isEditable );
```

Comment:
TIL `.toHaveBeenCalledWith()` and `.toBeCalledWith()` are actual aliases. https://jestjs.io/docs/en/expect#tohavebeencalledwitharg1-arg2-
Knowing that I like to former better as it seems grammatically correct when asserting after the fact.